tests/unittests/libs/test_database.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pony import orm
+
+
+@pytest.fixture
+def fake_jira_issue():
+    return {
+        "issue_key": "TEST-1234",
+        "title": "Test Issue",
+        "url": "https://jira.com",
+        "project": "TEST",
+        "severity": "blocker",
+        "status": "Open",
+        "customer_escaped": False,
+        "date_created": "2024-01-01",
+        "last_updated": "2024-01-01",
+    }
+
+
+@pytest.mark.usefixtures("temp_sqlite_db")
+@orm.db_session
+def test_database_services_entry(temp_sqlite_db):
+    test_service = temp_sqlite_db.Services(name="test-service-entry-service")
```

Comment:
setup steps should be in fixtures.
you can have `service` fixtures which accepts name through request.param and use it in `temp_sqlite_db.Services(name=request.param)`